src/sdl/sdl2graphics.cpp

```diff
@@ -381,10 +382,10 @@ namespace gcn
         
     void SDL2Graphics::drawRectangle(const Rectangle& rectangle)
     {
-        int x1 = rectangle.x;
-        int x2 = rectangle.x + rectangle.width - 1;
-        int y1 = rectangle.y;
-        int y2 = rectangle.y + rectangle.height - 1;
+        const int x1 = rectangle.x;
+        const int x2 = rectangle.x + rectangle.width - 1;
+        const int y1 = rectangle.y;
+	const int y2 = rectangle.y + rectangle.height - 1;
```

Comment:
You used tab instead of space...
We probably should add/commit a .clang-format

Reply:
Ah, that's right. I wanted to discuss that, as my VS automatically adds tabs, but it seems many/most of the upstream files use spaces. Either way is fine with me, but we should stick to one or the other :)